### DISCUSSION TITLE
Polycrystal with void

### DISCUSSION THREAD

**Discussion by inandi2** [neperfepx/neper#601] [Q&A] [answered]
Hi Quey,
Is it possible to generate a polycrystal with void inside of it like the attached image?

![image](https://user-images.githubusercontent.com/61553917/200346689-b0608040-425a-4ec8-9ce7-739bca4b9925.png)
Thank you,
Indrajit

**Comment by rquey** [comment, marked as answer]
No, but you can easily do it through some postprocessing.  Assuming you are using a tesr, you can define voids by setting the voxel cell ids to 0.

**Reply by smallrain1996** [reply]
https://doi.org/10.1016/j.engfracmech.2022.108332
This should be a successful case, but it is not clear how to realize this kind of hole.
